src/app/portlets/dot-edit-page/content/services/html/iframe-edit-mode.js.ts

```diff
@@ -90,6 +90,8 @@ export const EDIT_PAGE_JS = `
             });
         }
     })
+
+    window.getDotNgModel = getDotNgModel;
```

Comment:
I remove it but we need it [here](https://github.com/dotCMS/core-web/blob/master/src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts#L187)

Reply:
Also I tried to move this to the angular side, but we use it [here](https://github.com/dotCMS/core-web/blob/master/src/app/portlets/dot-edit-page/content/services/html/iframe-edit-mode.js.ts#L74) to send the model down with the event.